Sources/SourceKitLSP/SourceKitServer.swift

```diff
@@ -288,7 +321,8 @@ public final class SourceKitServer: LanguageServer {
     }
   }
 
-  func languageService(for uri: DocumentURI, _ language: Language, in workspace: Workspace) -> ToolchainLanguageServer? {
+  /// **Public for testing purposes only**
```

Comment:
I refrained from the underscore since it's also called internally, which is totally valid. I've decided to create a public underscored wrapper for the public test usage.